docs/upm_readme.md

```diff
@@ -139,6 +140,8 @@ If you would like to see specific examples of various EOS features in action, im
 The included samples show fully functional feature implementation that will both help with EOS integration as well as validate client to dev portal configuration. After installing the samples from the UPM package, you will find scenes for each major feature.
 
 <img src="images/sample_screen_lobby.gif" alt="Lobby Screenshot" width="48%"/>
+<a href="/readme.md"><img src="/docs/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
```

Comment:
Is there supposed to be a PEW logo here?

Reply:
yep, good catch!